akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -751,6 +752,30 @@ Host: example.com
 
       Await.result(binding.unbind(), 10.seconds)
     }
+
+    "produce a useful error message when connecting to a HTTPS endpoint over HTTP" in Utils.assertAllStagesStopped {
+      // akka/akka-http#1219
```

Comment:
Isn't this about https://github.com/akka/akka-http/issues/2614 ?

Reply:
Not sure what I was thinking
```suggestion
      // akka/akka-http#2614
```